contracts/script/deploy/certverifier/README.md

```diff
@@ -40,3 +40,13 @@ The deployment will output the address of the deployed contract to a json file i
     "eigenDACertVerifier": "0x..."
 }
 ```
+
+#### EigenDA V1
+
+To deploy with just EigenDA V1 verification enabled, you only need to pass valid addresses for the following dependency contracts into your json config:
+- `eigenDAThresholdRegistry`
+- `eigenDAServiceManager`
```

Comment:
registryCoordinator and operatorStateRetriever are v1 contracts yet they are set to 0x00 in your `mainnet.v1.config.json`.